drivers/i2c/i2c_npcx_controller.c

```diff
@@ -203,6 +204,8 @@ struct i2c_ctrl_data {
 #define HAL_I2C_INSTANCE(dev)                                                                      \
 	((struct smb_reg *)((const struct i2c_ctrl_config *)(dev)->config)->base)
 
+#define DEV_IS_I2C1(dev) (((uintptr_t)HAL_I2C_INSTANCE(dev)) == ((uintptr_t)0x4000b000))
+
```

Comment:
This define is not longer used.
You also need to drop the `i2c1_0` from the commit message as this now affects behavior on all buses.